doc/source/cluster/cluster_under_construction/ray-clusters-on-kubernetes/examples/ml-example.md

```diff
@@ -1,10 +1,253 @@
 (kuberay-ml-example)=
 
-# Example machine learning workloads
+# XGBoost-Ray on Kubernetes
 
-:::{warning}
-This page is under construction!
+:::{note}
+To learn the basics of Ray on Kubernetes, we recommend taking a look
+at the {ref}`introductory guide<kuberay-quickstart>` first.
 :::
 
-At least one end-to-end example of an actual machine learning workload,
-preferably with GPUs, possibly engaging the autoscaling functionality.
+In this guide, we show you how to run a sample Ray machine learning
+workload on Kubernetes infrastructure.
+
+We will run Ray's {ref}`XGBoost training benchmark<xgboost-benchmark>` with a 100 gigabyte training set.
+* Learn more about {ref}`XGBoost-Ray<xgboost-ray>`.
+
+## A note on autoscaling
```

Comment:
TODO: Move this discussion into a doc page dedicated to autoscaling and link there.

Reply:
Yes, this should be moved -- for this section, the user just wants to run the workload and not think too deeply about autoscaling, so we should just provide them with something that works. By default probably with autoscaling support since in general we want that to work well and be the standard way of deploying Ray clusters.

Reply:
I agree it should work well. I disagree that autoscaling should be on by default (for any distributed system).

I actually think this kind of batch workload is great for the new RayJob controller.